src/schemas/ProgramDonorSummary/resolvers.ts

```diff
@@ -0,0 +1,95 @@
+import { IResolvers } from 'apollo-server-express';
+import { GlobalGqlContext } from 'app';
+import { GraphQLFieldResolver, GraphQLObjectType } from 'graphql';
+import { DonorSummaryEntry, ProgramDonorSummaryStats, ProgramDonorSummaryFilter } from './types';
+
+const programDonorSummaryEntriesResolver: GraphQLFieldResolver<
+  unknown,
+  GlobalGqlContext,
+  {
+    programId: string;
+    first: number;
+    last: number;
+    filters: ProgramDonorSummaryFilter[];
+  }
+> = (source, args, context): DonorSummaryEntry[] => {
+  const { programId } = args;
+
+  console.log('args: ', args.filters);
+
+  return [];
+};
+
+const programDonorSummaryVersionResolver: GraphQLFieldResolver<
```

Comment:
Not sure I understand where this will come from in es?

Reply:
after a discussion, we decided this isn't necessary yet :) will spec out later as need arises